### PR TITLE
Chore: Added Frontend Mentor

### LINKED ISSUE #990: [Add] Frontend Mentor
### Site URL

https://www.frontendmentor.io/

### Description of the above provided site

Solve real-world HTML, CSS and JavaScript challenges whilst working to professional designs. Join 632,729 developers building projects, reviewing code, and helping each other get better.

### Category

Frontend

### Sub Category

UI Generator

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by parma1111**
## Fixes Issue

Closes #990 

## Changes proposed

Solve real-world HTML, CSS and JavaScript challenges whilst working to professional designs. Join 632,729 developers building projects, reviewing code, and helping each other get better.

## Screenshots

Screenshot (824)
1.) Updated-description
Screenshot (825)
2.) Updated-description
Screenshot (826)

## Note to reviewers

Any kind of feedback would be appreciated.

### CODE REVIEW

**Review by rupali-codes — CHANGES_REQUESTED** [review]
Please update the description to what the site actually provides :)

**Comment by parma1111** [comment]
> Please update the description to what the site actually provides :)

> Hey @rupali-codes :) It's done, now review the PR.

File: database/frontend/ui-generators.json [hunk 1]
```diff
@@ -111,6 +111,13 @@
     "category": "frontend",
     "subcategory": "ui-generators"
   },
+  {
+    "name": "Frontend Mentor",
+    "description": "This site solve real-world HTML, CSS and JavaScript challenges whilst working to professional designs.",
```
Comment (CBID2) [inline database/frontend/ui-generators.json:116]: [Proofreading]
```suggestion
    "description": "This site solve real-world HTML, CSS, and JavaScript challenges whilst working to professional designs.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @parma1111! :) I made a suggestion on how to one of your descriptions.

File: database/frontend/ui-generators.json (hunk 1 above)
Comment (parma1111) [reply database/frontend/ui-generators.json]: Hey @CBID2 :) done , now review the PR

**Review by CBID2 — APPROVED** [review]
LGTM!

**Review by ujjawaltyagii — APPROVED** [review]
lgtm !

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**